dataactcore/read_config.py

```diff
@@ -29,18 +31,9 @@
 # set the location of the Alembic config file
 ALEMBIC_PATH = os.path.join(dirname(abspath(__file__)), 'alembic.ini')
 MIGRATION_PATH = os.path.join(dirname(abspath(__file__)), 'migrations')
-CONFIG_BROKER = {}
-CONFIG_SERVICES = {}
-CONFIG_DB = {}
-CONFIG_LOGGING = {}
-CONFIG_JOB_QUEUE = {}
-
-def merge_config(existing_config, new_config):
-    if type(existing_config) is not dict or type(new_config) is not dict:
-        raise ValueError("Both args to merge_config must be dicts")
-    for key in new_config.keys():
-        existing_config[key] = new_config[key]
-    return existing_config
+
+for value in CONFIG_CATEGORIES.values():
+    locals()[value] = {}
```

Comment:
So CONFIG_CATEGORIES is holding the names of the various config dictionaries, this is a way of creating the variable with the name stored in CONFIG_CATEGORIES.  If there's a better way to do that, please let me know, I'm also using this down in the logging section to get these variables.  If this were within a class I'd be using setattr(), is there a more direct equivalent of that for a current namespace?